### DISCUSSION TITLE
exclude pod or namespace or containers from Kubernates logs

### DISCUSSION THREAD

**Discussion by therus000** [vectordotdev/vector#16519] [Q&A] [answered]
Good day!
I just read the documentation but didn't understand how to do that 
may be someone can help me .
I'm using vector to collect Kubernetes logs. My config by default in configmap.yaml
in source ad kubernetes_logs
how can i exclude logs some containers or namespace ro pods?

**Comment by StephenWakely** [comment, marked as answer]
Have a look at the docs here: https://master.vector.dev/docs/reference/configuration/sources/kubernetes_logs/#filtering

Generally it would be preferable to set a label on the namespace `vector.dev/exclude: "true"`. These will then automatically be excluded from the source.

Let me know if you have further questions.
